crates/config-encryption/todo.md

```diff
@@ -0,0 +1,9 @@
+
+
+- TS Client
+- OpenAPI spec
+- Prometheus metrics
+- CORS headers
+- Tests
+- Build image
+- Deploy
```

Comment:
Did you mean to leave this file checked in?

Reply:
lol _totally_ ... I'll remove. Thanks for catching that.